site/styles/input.css

```
@@ -0,0 +1,18 @@
+@import 'tailwindcss';
+
+@source "../**/*.html";
```

Stylelint (17.9.0)

[error] Unexpected unknown at-rule "@source" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)